lib/mutant/ast/regexp/transformer.rb

```diff
@@ -0,0 +1,82 @@
+module Mutant
+  module AST
+    module Regexp
+      class Transformer
+        include AbstractType
+
+        REGISTRY = {} # rubocop:disable MutableConstant
```

Comment:
Will do. I have some local code which extracts a generic `Mutant::TypeRegistry` which I'll insert here (shared behavior with `Mutant::Mutator::Registry`) and I'll probably implement the finalize there